src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>)}
+    {person.isMarried && person.sex === 'm' ? (
+      <p className="Person__partner">{`${person.partnerName} is my wife`}</p>
+    ) : ''}
+    {person.isMarried && person.sex === 'f' ? (
+      <p className="Person__partner">{`${person.partnerName} is my husband`}</p>
+    ) : ''}
+    {person.isMarried === false ? (
+      <p className="Person__partner">I am not married</p>
+    ) : ''}
```

Comment:
```suggestion
   const partner = person.sex === 'm' ? `${person.partnerName} is my husband` : `${person.partnerName} is my wife` 
   .....
    <p className="Person__partner">
       {person.isMarried 
         ? partner
         : 'I am not married'
    </p>
```

Reply:
create a variable for partner text (before jsx)

then use ternary operator to display it or 'not married'